src/llamacrawl/services/readers/swag.py

```
@@ -178,6 +182,9 @@ def _list_config_files(self) -> list[tuple[str, str]]:
             # Remote listing via SSH
             self._connect_ssh()
             if self.ssh_client:
+                stdin: ChannelStdinFile
+                stdout: ChannelFile
+                stderr: ChannelStderrFile
                 stdin, stdout, stderr = self.ssh_client.exec_command(
```

Ruff (0.14.0)

Unpacked variable `stdin` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)